config/tinydns/tinydns.xml

```diff
@@ -2,45 +2,47 @@
 <!DOCTYPE packagegui SYSTEM "../schema/packages.dtd">
 <?xml-stylesheet type="text/xsl" href="../xsl/package.xsl"?>
 <packagegui>
-        <copyright>
-        <![CDATA[
+	<copyright>
+	<![CDATA[
 /* $Id$ */
-/* ========================================================================== */
+/* ====================================================================================== */
 /*
-    tinydns.xml
-    part of pfSense (http://www.pfSense.com)
-    Copyright (C) 2007 to whom it may belong
-    All rights reserved.
-                                                                              */
-/* ========================================================================== */
+	tinydns.xml
+	part of pfSense (https://www.pfSense.org/)
+	Copyright (C) 2009 Bill Marquette
+	Copyright (C) 2010 Ermal Luçi
+	Copyright (C) 2010-2011 Scott Ullrich
+	Copyright (C) 2012-2015 ESF, LLC
```

Comment:
Again, I don't have anything else that the commits history. Considering it was completely missing until now, I figured it's better than leaving it empty. ;)